### ISSUE TITLE
Error when loading .bootstraprc with missing `scripts` section

### ISSUE THREAD

**Issue by kkamperschroer** [shakacode/bootstrap-loader#152]
Hello!

With the release of 1.1.2, I've encountered an issue loading my .bootstraprc file unless I include an empty `scripts` section. I haven't had a need for the `scripts` section and only had a `styles` section. It does not seem to matter if the config is json or yaml.

An example of my offending config:
```
{
  "bootstrapVersion": 3,
  "preBootstrapCustomizations": "./styles/_variables.scss",
  "appStyles": "./styles/_bootswatch.scss",
  "styleLoaders": [
    "style",
    "css",
    "sass"
  ],
  "env": {
    "development": {
      "extractStyles": false
    },
    "production": {
      "extractStyles": true
    }
  },
  "styles": {
    "mixins": true,
    "normalize": true,
    "print": true,
    "scaffolding": true,
    "type": true,
    "code": true,
    "grid": true,
    "tables": true,
    "forms": true,
    "buttons": true
  }
}
```

And the corresponding working version:
```
{
  "bootstrapVersion": 3,
  "preBootstrapCustomizations": "./styles/_variables.scss",
  "appStyles": "./styles/_bootswatch.scss",
  "styleLoaders": [
    "style",
    "css",
    "sass"
  ],
  "env": {
    "development": {
      "extractStyles": false
    },
    "production": {
      "extractStyles": true
    }
  },
  "styles": {
    "mixins": true,
    "normalize": true,
    "print": true,
    "scaffolding": true,
    "type": true,
    "code": true,
    "grid": true,
    "tables": true,
    "forms": true,
    "buttons": true
  },
  "scripts": {}
}
```
Note the empty `scripts` section.

The stack trace from the bad config:
```
Error: ./~/bootstrap-loader/lib/bootstrap.loader.js!./~/bootstrap-loader/no-op.js
Module build failed: TypeError: Cannot convert undefined or null to object
    at Function.keys (native)
    at exports.default (/home/ubuntu/fons-app/node_modules/bootstrap-loader/lib/utils/selectModules.js:14:17)
    at createConfig (/home/ubuntu/fons-app/node_modules/bootstrap-loader/lib/bootstrap.config.js:125:44)
    at Object.module.exports.pitch (/home/ubuntu/fons-app/node_modules/bootstrap-loader/lib/bootstrap.loader.js:100:40)
 @ ./~/bootstrap-loader/loader.js 1:0-44
```

So the thing that is undefined or null is the scripts section of the config, and then it tries to do `Object.keys` on that, which obviously fails.

Let me know if you need more information! Thanks for creating such an awesome module!

**Comment by justin808** [comment]
@kkamperschroer Can you please try 1.1.3?

**Comment by justin808** [comment]
See #154.

**Comment by justin808** [comment]
Please confirm this is fixed in 1.1.4.

**Comment by kkamperschroer** [comment]
Looks like it is fixed in 1.1.4. Thank you!
